sdk/identity/Azure.Identity/src/Credentials/AzurePipelinesCredentialOptions.cs

```diff
@@ -40,6 +40,11 @@ public class AzurePipelinesCredentialOptions : TokenCredentialOptions, ISupports
         /// </summary>
         internal string TeamProjectId { get; set; } = Environment.GetEnvironmentVariable("SYSTEM_TEAMPROJECTID");
 
+        /// <summary>
+        /// The hub under which this pipeline is running - typically "build" or "release".
+        /// </summary>
+        internal string HubName { get; set; } = Environment.GetEnvironmentVariable("SYSTEM_HOSTTYPE");
```

Comment:
Would you mind updating the `AzurePipelinesCredentialTests.AzurePipelinesCredentialOptions_Loads_From_Env` test to validate this environment variable also? Thanks!

Reply:
I've added an assert to the test.